src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -544,6 +578,304 @@ private static unsafe void DispatchTailCalls(
             }
         }
 
+        private static Continuation AllocContinuation(Continuation prevContinuation, nuint numGCRefs, nuint dataSize)
```

Comment:
All this should be in separate file (AsyncHelpers or whatever the API review decides.)

Reply:
I think a good chance it will be `AsyncHelpers`

Reply:
Moved to `AsyncHelpers`, which did end up the container for public API after the review.